studio/schemas/documents/companyInfo.ts

```diff
@@ -0,0 +1,131 @@
+import { defineType, defineField } from "sanity";
```

Comment:
The scope for this PR was to migrate to a new document type with a different name. Do you want to increase the scope? If so, do you have any suggestions?

Reply:
Yes, I'd love to add this to current PR. It's relevant because we're not just renaming we're creating a new schema. 

If you want to be more generic/specific you can adjust title to "remove SiteSettings and create CompanyInfo"

Reply:
Ah, I meant suggestions for the schema title and descriptions 🙃, that's what you wanted to update right?

Reply:
hahah, sorry my bad. 

but yeah, only suggestions to show what I meant.